src/reports.cpp

```diff
@@ -1575,6 +1575,26 @@ REPORT_GENERATOR(terrain_info, rc)
 			continue;
 		blit_tced_icon(cfg, terrain_id, terrain_icon, high_res, terrain_name);
 	}
+
+	if (map.is_village(mouseover_hex)) {
```

Comment:
This file doesn't seem to have been updated, but I think our style is `if(` without the space.